Apps/Examples/Examples/All Examples/LiveDataExample.swift

```diff
@@ -0,0 +1,96 @@
+import MapboxMaps
+import Turf
+
+@objc(LiveDataExample)
+
+class LiveDataExample: UIViewController, ExampleProtocol {
```

Comment:
```suggestion
@objc(LiveDataExample)
class LiveDataExample: UIViewController, ExampleProtocol {
```

Removing the empty line makes it more clear that the annotation modifies the class